### ISSUE TITLE
Missing dependency: @angular-devkit/core

### ISSUE THREAD

**Issue by nfriend** [csotomon/Angular2-AdminLTE#44]
What I did:

- Cloned this repo: 
  - `git clone https://github.com/csotomon/Angular2-AdminLTE.git`
- Installed dependencies:
  - `cd Angular2-AdminLTE`
  - `npm install`
- Ran the build:
  - `ng serve`

Error message:

```bash
Error: Cannot find module '@angular-devkit/core'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\nfriend\Documents\GitHub\Angular2-AdminLTE\node_modules\@angular-devkit\schematics\src\tree\virtual.js:10:16)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)

```

Installing the dependency manually (`npm install @angular-devkit/core --save`) fixes the issue.

**Comment by csotomon** [comment]
Hi, than you for report this bug. I will fixed it

**Comment by csotomon** [comment]
This is solved in version 3.0.0
